api/v1/infinispan_types.go

```diff
@@ -611,6 +611,9 @@ type InfinispanStatus struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Operator Status"
 	Operator Operator `json:"operator,omitempty"`
+	// The Operator Selector
+	// +optional
+	Selector string  `json:"selector"`
```

Comment:
Why do we need this additional field?

Reply:
The HPA that KEDA generates requires the target CR to have a selector.
> The HPA will use the scale endpoint to retrieve the labelselectors of the child pods, as well as the status and spec of replicas for scaling.

I basically followed this article: https://medium.com/@thescott111/autoscaling-kubernetes-custom-resource-using-the-hpa-957d00bb7993

Reply:
Thanks for linking the article, that was really helpful :+1:

In that case I'm happy to add the additional field. Can you update the description of the field to be more informative, maybe:

"The Selector used to identify Infinispan cluster pods"